bitcoin/src/blockdata/script/witness_program.rs

```diff
@@ -130,10 +147,12 @@ impl fmt::Display for Error {
         use Error::*;
 
         match *self {
-            InvalidLength(len) =>
-                write!(f, "witness program must be between 2 and 40 bytes: length={}", len),
-            InvalidSegwitV0Length(len) =>
-                write!(f, "a v0 witness program must be either 20 or 32 bytes: length={}", len),
+            InvalidLength(len) => {
+                write!(f, "witness program must be between 2 and 40 bytes: length={}", len)
+            }
+            InvalidSegwitV0Length(len) => {
+                write!(f, "a v0 witness program must be either 20 or 32 bytes: length={}", len)
+            }
```

Comment:
These changes are not needed, worse still, they will be changed back by the weekly `rustfmt` job we have. (I'm not 100% sure how these changes got into your PR but FTR we run `rustfmt` with `nightly`, perhaps stable `rustfmt` removed them?).

Reply:
i'm not sure what went wrong here, but i'm definitely formatting my code with nightly. 

```
$ rustfmt --check --config-path rustfmt.toml bitcoin/src/blockdata/script/witness_program.rs 
$ rustfmt --version
rustfmt 1.7.0-nightly (2d7be73 2023-12-23)
```

Maybe there was a change in a recent version of nightly `rustfmt` that hasn't been applied via your CI yet?

Reply:
I usually run: `cargo +nightly fmt`

Reply:
I'm not sure why you are targeting a specific file with rustfmt.

Reply:
Ah, looks like `cargo fmt` will update all files while `rustfmt` requires a list of target files.

Reply:
> Maybe there was a change in a recent version of nightly rustfmt that hasn't been applied via your CI yet?

Well I don't think the CI version is in question, but rather the weekly rustfmt job.  Your nightly might be newer right now, but maybe not once it runs for the week.

Reply:
OK, i'll remove this change and let the CI take care of updating the formatting later

Reply:
It seems to me like it's best to just leave the formatting that way there won't be a future commit needed by the weekly format bot.